### ISSUE TITLE
Add Sepis

### ISSUE THREAD

**Issue by thomas15v** [thomas15v/RealMedicine#4]
Well I learned something new when looking for things to replace infection deaths. Instead of dying; pawns will go in to [Septic shock ](https://en.wikipedia.org/wiki/Septic_shock) (and wish that they have died).

Septic progress starts when infection reaches 75% severity. When Sepsis reached 15% severity the pawn will get sick and it won't go down naturally anymore.

EDIT: When Sepsis gets it's trigger percentage (15%), there will be a chance that flu, heart attack (cardiac arrest) or organ damage occurs, instead of the full deseas. 

Sepis will cause the following:

- Septic Shock
  - Pain: +80%
  - Increased chance of heart attack and random organ failure
  - Downed
  - Moments of passing out and screaming in pain (Screaming mechanic)
- Severity 100%
  - Death

Sepsis can be treated with:
- Antibiotics (A dose every 2 hours)
- Luciferium
- Sanoferium

**Comment by Avexsis** [comment]
Here I want to add that sepsis can't be just sepsis and severe sepsis. Sepsis is always extremely severe, it can't have any grades of "severiness". And always 100% lethal unless on high-tech. You can't sit it out. Its an infection spreading with blood through the whole body. 
A feature for this could be extremely high antibiotic doses, like every 2 hours of so. Thats actually how it is treated in real life, they monitor the infection every hour because it changes, reaches new organs and requires antibiotics change. Skip an hour and a patient is dead.

It should start with the septic shock straightaway and uncurable on low-tech. Also could be a result for a failed surgery and of course a dirty hospital.
Hope it helps.

**Comment by thomas15v** [comment]
This definitely helps. I will make it go straight to septic shock with all the negative side effects and a Antibiotics cure every 2 hours.

I do have some questions tho:
- Is it correct that untreated/Badly treated infections always cause Sepis?
- If an organ shuts down, what would be the most realistic way to get it back working or keep the patient alive? (I would prefer 1 or 2 generic solutions)

**Comment by Avexsis** [comment]
No. Infections are various and not all of them cause sepsis. Some can end in a disease like AIDS, hepatitis, tuberculosis, gangren, malaria or flu. Or just cause pain and swelling and be cured with herbal medicine. Like a bite or scratch from an animal, can cause rabies (uncurable) or just be tended with alcohol and be forgotten in a couple of days. 
Thats why I miss "easy" infections that would require a single treatment with herbal medicine. 

Shuts down means dies? A dead organ can't be revived,  a patient either gets a new organ transplanted (and a life-long medicine dependance and often another transpantation after some years or even months) or lives with one organ or without (depends on organ). For example a person can live with one kidney, or even without kidneys at all (regularly attending to a special procedure) but threre is a very high chance that such life would be shortened to like 10 years at max because it has side-effects. 
Works not with all organs, for example a pancreas or hepat are vital organs.
